eZ/Bundle/EzPublishRestBundle/Features/Context/SubContext/Views.php

```diff
@@ -34,4 +36,16 @@ public function theSearchHitsAreContentObjects()
             Assertion::assertInstanceOf('eZ\Publish\API\Repository\Values\Content\Content', $searchHit->valueObject);
         }
     }
+
+    /**
+     * @Given /^I set contentQuery.Filter to a set of Criterion objects$/
```

Comment:
nitpick: Shouldn't  `filter` be a parameter, sentence implies several criteria when there is only one, but maybe stick with singular and let sentance specify it so sentence is more clearly reflecting what is being done?